packages/cli/templates/webcomponents/igc-ts/projects/_base/files/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en-GB">
+
+<head>
+  <meta charset="utf-8">
+  <base href="/">
+  <title>Ignite UI for Web Components</title>
```

Comment:
Fairly certain the title is supposed to be templated from the user-provided one, no?

Reply:
I was using the Ignite UI for Angular template as a guide so I think it is fine the way it is now?

Reply:
Hah, could very well be wrong in Angular's templates too :) 
I guess it's a preference, though kind of odd you give a name for the app and that's not in the title, right? Can be left for a separate item as well